README.md

```diff
@@ -18,59 +29,59 @@ If you're new here, here are some things you should know:
 
-## Development
+### Install Postgres
+You must install postgres and run the database locally. Instructions differ depending on the operating system.
 
-### Installation Instructions
+Follow one of these guides to install postgres:
 
-The `installation.md` file ([https://github.com/rubyforgood/human-essentials/blob/main/installation.md](https://github.com/rubyforgood/human-essentials/blob/main/installation.md)) has detailed instructions for installation and configuration of an Ubuntu host to run this software. Although there is not a document for Mac OS, it may be helpful for that as well.
+###### MacOSX
+https://www.digitalocean.com/community/tutorials/how-to-use-postgresql-with-your-ruby-on-rails-application-on-macos
 
-### Ruby Version
-This app uses Ruby version 3.0.2, indicated in `/.ruby-version` and `Gemfile`, which will be auto-selected if you use a Ruby versioning manager like `rvm`, `rbenv`, or `asdf`.
+###### Ubuntu
+TODO
 
-### Yarn Installation
-If you don't have Yarn installed, you can install with Homebrew on macOS `brew install yarn` or visit [https://yarnpkg.com/en/docs/install](https://yarnpkg.com/en/docs/install). Be sure to run `yarn install` after installing Yarn. NOTE: It's possible that Node version 12 may cause you problems, see issue #751. Node 10 or 11 seem to be fine.
+###### Windows
+TODO
 
-### Install dependencies using Yarn
-Run `yarn` to install project dependencies.
+After completing this step, you should have both ruby, node, and postgres installed. Now you can proceed to the next step which is to setup the application.
 
-### Create your .env with database credentials
-Be sure to create a `.env` file in the root of the app that includes the following lines (change to whatever is appropriate for your system):
+### Run Setup
+Run the following command should run all the neccessary steps to get up and running.
 ```
-PG_USERNAME=username
-PG_PASSWORD=password
+bin/setup
 ```
-If you're getting the error `PG::ConnectionBad: fe_sendauth: no password supplied`, it's because you have probably not done this.
-
-### Database Configuration
-This app uses PostgreSQL for all environments. You'll also need to create the `dev` and `test` databases, the app is expecting them to be named `diaper_dev`, `diaper_test`, `partner_dev`, and `partner_test` respectively. This should all be handled with `rails db:setup`.
-Create a `database.yml` file on `config/` directory with your database configurations. You can also copy the existing file called `database.yml.example` as an example and just change the credentials.
 
-### Seed the database
-From the root of the app, run `bundle exec rails db:seed`. This will create some initial data to use while testing the app and developing new features, including setting up the default user.
+### Run the application
+Run the following command to start the server locally.
+```
+bin/start
+```
```

Comment:
Got the following errors after running bin/start
![error](https://user-images.githubusercontent.com/14355495/173702064-13ec52de-3c91-4e30-859e-5c14b09abbac.png)
<img width="1792" alt="Screen Shot 2022-06-14 at 18 48 08" src="https://user-images.githubusercontent.com/14355495/173702074-65c7d226-890b-4810-a4d3-7abc6436a252.png">

Reply:
This solved it https://github.com/rails/webpacker/issues/2825#issuecomment-764888322

Reply:
Do you think adding that `rails webpacker:compile` step in the `bin/setup` would work?

Reply:
Hopefully that will work. I'll give it a shot and let you know how it went